[0m15:00:25.542641 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1145fa8f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1157aa650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1157a9f30>]}


============================== 15:00:25.544971 | 84f0cfae-fc71-40df-9b40-17cf9c687955 ==============================
[0m15:00:25.544971 [info ] [MainThread]: Running with dbt=1.9.0
[0m15:00:25.545323 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/savana/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:01:26.551369 [debug] [MainThread]: Starter project path: /Users/savana/Desktop/BokksuV2/dbt-persona-project/venv/lib/python3.10/site-packages/dbt/include/starter_project
[0m15:01:26.555551 [info ] [MainThread]: 
Your new dbt project "dbt_persona" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m15:01:26.555844 [info ] [MainThread]: Setting up your profile.
[0m15:29:48.699433 [info ] [MainThread]: Profile dbt_persona written to /Users/savana/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m15:29:48.703577 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 1763.224, "process_in_blocks": "0", "process_kernel_time": 0.130812, "process_mem_max_rss": "123224064", "process_out_blocks": "0", "process_user_time": 0.790412}
[0m15:29:48.704223 [debug] [MainThread]: Command `dbt init` succeeded at 15:29:48.704083 after 1763.22 seconds
[0m15:29:48.704641 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1145fa8f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11571e2f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1157ab1c0>]}
[0m15:29:48.705055 [debug] [MainThread]: Flushing usage events
[0m15:29:49.011904 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m13:12:19.027414 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1039eebc0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1054a87f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1054a80d0>]}


============================== 13:12:19.029917 | 9ee6ed22-26ee-4be3-9adb-3c4b27b0498e ==============================
[0m13:12:19.029917 [info ] [MainThread]: Running with dbt=1.9.0
[0m13:12:19.030209 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'profiles/', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt run --select my_first_dbt_model', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m13:12:19.030530 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/savana/Desktop/BokksuV2/dbt-persona-project/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m13:12:19.034276 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": false, "command_wall_clock_time": 0.043595668, "process_in_blocks": "0", "process_kernel_time": 0.105033, "process_mem_max_rss": "93421568", "process_out_blocks": "0", "process_user_time": 0.53176}
[0m13:12:19.034533 [debug] [MainThread]: Command `dbt run` failed at 13:12:19.034484 after 0.04 seconds
[0m13:12:19.034718 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1039eebc0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103e568c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1054a9e10>]}
[0m13:12:19.034887 [debug] [MainThread]: Flushing usage events
[0m13:12:19.388806 [debug] [MainThread]: An error was encountered while trying to flush usage events
